docs/explanation/reader-explanation.md

```diff
@@ -0,0 +1,274 @@
+                                   "/dat_mat_components/LI Demod 2 Y/value",
+                                   "/dat_mat_components/LI Demod 2 Y/unit"],
+                              "2": ["/dat_mat_components/Bias [filt]/value",
+                                   "/dat_mat_components/Bias [filt]/unit",
+                                   "dat_mat_components/Bias [filt]/metadata",
+                                   "/dat_mat_components/Current [filt]/value",
+                                   "/dat_mat_components/Current [filt]/unit",
+                                   "/dat_mat_components/Current [filt]/metadata",
+                                   "/dat_mat_components/Temperature 1 [filt]/value",
+                                   "/dat_mat_components/Temperature 1 [filt]/unit",
+                                   "/dat_mat_components/Temperature 1 [filt]/metadata",
+                                   "/dat_mat_components/LI Demod 1 X [filt]/value",
+                                   "/dat_mat_components/LI Demod 1 X [filt]/unit",
+                                   "/dat_mat_components/LI Demod 1 X [filt]/metadata",
+                                   "/dat_mat_components/LI Demod 1 Y [filt]/value",
+                                   "/dat_mat_components/LI Demod 1 Y [filt]/unit",
+                                   "/dat_mat_components/LI Demod 1 Y [filt]/metadata",
+                                   "/dat_mat_components/LI Demod 2 X [filt]/value",
+                                   "/dat_mat_components/LI Demod 2 X [filt]/unit",
+                                   "/dat_mat_components/LI Demod 2 X [filt]/metadata",
+                                   "/dat_mat_components/LI Demod 2 Y [filt]/value",
+                                   "/dat_mat_components/LI Demod 2 Y [filt]/unit",
+                                   "/dat_mat_components/LI Demod 2 Y [filt]/metadata"]}
+}
+```
+**NOTES**
+
+- Each key is nevigating the NeXus concept (e.g. `/ENTRY[entry]/INSTRUMENT[instrument]/piezo_config/active_calib` key nevigates `ENTRY` -> `INSTRUMENT` -> `piezo_config` -> `active_calib` field in `NXsts` application definition.) in NeXus application definition.  
+- Value denoted by `@eln` refers the data must come from eln (user provided), but can be changed if rwa file contains that piece of data. 
+- To update (if needed) the config file, a set of rules needs to be followed:
+  - The dictionary in config files have the following meaning:
+    ```
+    "/ENTRY[entry]/INSTRUMENT[instrument]/lock_in/harmonic_order_N": {"D1": {"value": "/Lock-in/Harmonic D1/value"},
+                                                                      "D2": {"value": "/Lock-in/Harmonic D2/value"}},
+    ```
+    Here, the part `N` in field `harmonic_order_N`, can be considered as the name of dimensions, can be replaced by `D1` and `D2` to  write two `harmonic_order` and can be extended for further dimensions.
```

Comment:
```suggestion
    Here, the part `N` in field `harmonic_order_N` can be considered as the name of dimensions and can be replaced by `D1` and `D2` to  write two fields of `harmonic_order` . This can be extended to further dimensions.
```

Reply:
Resolved!